src/Uno.UI.RuntimeTests/Tests/Uno_UI_Services/Given_ResourceService.cs

```diff
@@ -1,21 +1,37 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿#if __ANDROID__ || __IOS__
 using System;
 using System.Collections.Generic;
 using System.Text;
 using Windows.UI.Xaml;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Uno.UI.RuntimeTests.Tests.Uno_UI_Services
 {
-#if __IOS__ || __ANDROID__
 	[TestClass]
 	public class Given_ResourceService
 	{
+		private IResourcesService _sut;
+
+		[TestInitialize]
+		public void Init()
+		{
+			_sut = ResourceHelper.ResourcesService;
+		}
+
 		[TestMethod]
 		public void When_Valid_NativeResource()
```

Comment:
You removed this test for .net/UWP...

Reply:
Indeed, but as it's runtime test they do run on those platforms ... BUT is also disabled this test on WASM which is not valid! Fixed !

Reply:
Well, actually the test was already targetting only iOS and Android and does not compile on other platforms (`ResourceHelper` is missing)... going to restore the restriction 😐